instrumental/drivers/funcgenerators/agilent.py

```diff
@@ -6,10 +6,15 @@
 MXG driver was initially developed for and tested on the N5181A.
 """
 from enum import Enum
+from pint import UnitRegistry
 from . import FunctionGenerator
 from .. import VisaMixin, SCPI_Facet
 from ... import u, Q_
 
+
+ureg = UnitRegistry()
```

Comment:
Instead of creating this, you can just use the already-imported `u`.

I'm not sure how recent versions of Pint may have changed, but this is important to ensure that all of the Instrumental code uses the same registry.